src/components/header.js

```diff
@@ -105,7 +103,7 @@ function Header({ slug }) {
       />
       <Menu
         key="one"
-        items={navDropdownItems1}
+        items={dropdowns[0]}
```

Comment:
I think nav bar items can be created by mapping through dropdowns, then concatting parentLinkItems as you already do. Here it's still hardcoded to 2 nav dropdown buttons

Reply:
Sorry saw that you said you'd work on this in another PR